src/Compilers/CSharp/Portable/Symbols/BaseTypeAnalysis.cs

```diff
@@ -219,10 +219,6 @@ private static (bool definitelyManaged, bool hasGenerics) DependsOnDefinitelyMan
             return (false, hasGenerics);
         }
 
-        /// <summary>
```

Comment:
>summary [](start = 13, length = 7)

nit: was something wrong with the comment (aside from Unset->Unknown disparity)?

Reply:
No, it just seemed like the kind of comment that's prone to go stale. I brought it back and updated it.